model/feature_flags.go

```diff
@@ -75,6 +75,8 @@ type FeatureFlags struct {
 
 	OnboardingTourTips bool
 
+	EnableCloudFreeDeprecationUI bool
```

Comment:
nit: simplify the name to be more clear on what the flag controls:
```suggestion
	DeprecateCloudFree bool
```